backend/python/app/services/implementations/comment_service.py

```diff
@@ -14,38 +17,75 @@ def __init__(self, logger=current_app.logger):
         self.logger = logger
 
     def create_comment(self, comment):
-        try:
-            new_comment = Comment(**comment)
-            new_comment.time = datetime.utcnow()
-            new_comment.resolved = False
-            comment_index = 0
-
-            max = (
-                db.session.query(func.max(Comment.comment_index))
-                .filter(
-                    Comment.story_translation_content_id
-                    == comment["story_translation_content_id"]
+        user_id = get_user_id_from_request()
+        new_comment = Comment(**comment)
+        new_comment.user_id = user_id
+
+        story_translation_content = StoryTranslationContent.query.filter_by(
```

Comment:
nice! this works, but getting _story_translation_content_ and getting _story_translation_ on line 29 are two database queries. Can you try using only 1 database query? You'll need to join `StoryTranslation` and `StoryTranslationContent` together.